src/annotator/config/test/index-test.js

```diff
@@ -99,6 +100,7 @@ describe('annotator.config.index', function() {
         branding: 'BRANDING_SETTING',
         openSidebar: 'OPEN_SIDEBAR_SETTING',
         requestConfigFromFrame: 'https://embedder.com',
+        configFrameAncestorLevel: 1,
```

Comment:
I think this change is left over from the first iteration of this PR and no longer applicable?